src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/JSON.txt

```diff
@@ -0,0 +1,79 @@
+// (UTC-08:00) Pacific Time (US & Canada)
+#SETUP: TimeZoneInfo("Pacific Standard Time")
+#SETUP: OptionSetTestSetup
+
+>> JSON(Blank())
+"null"
+
+>> JSON("hello")
+"""hello"""
+
+>> JSON(1.23)
+"1.23"
+
+>> JSON(0)
+"0"
+
+>> JSON(1)
+"1"
+
+>> JSON(-1)
+"-1"
+
+>> JSON(true)
+"true"
+
+>> JSON(false)
+"false"
+
+>> JSON(RGBA(10, 30, 50, 0.75))
+"""#0a1e32bf"""
+
+>> JSON(Color.Turquoise)
+"""#40e0d0ff"""
+
+>> JSON(OptionSet.Option2)
+"""option_2"""
+
+>> JSON(TimeUnit.Hours)
+"""hours"""
+
```

Comment:
This is not something we want people to do (and we should probably explicitly block it) - so we shouldn't "advertise" it in the tests (the _de facto_ specification for the function)

Reply:
Once the bug will be fixed we should be good
The Date will be converted to Local and back to UTC as expected

The plan is to have the same in French TZ and confirm it works

Reply:
Maybe not but that's a good way to explain the logic